stores/suggestionsStore/selectors.js

```diff
@@ -1,5 +1,9 @@
-export function getSuggestions(state) {
-  return state.suggestions;
+export function getSuggestions(state, count) {
+  if (count) {
+    return state.suggestions.slice(0, count);
+  } else {
+    return state.suggestions;
+  }
```

Comment:
This is alright, but I'd slice anyways, `slice(0, undefined)` doesn't do anything (just copies the array). 
```suggestion
    return state.suggestions.slice(0, count);
```